dev/RadioMenuFlyoutItem/RadioMenuFlyoutItem.idl

```diff
@@ -14,6 +14,12 @@ unsealed runtimeclass RadioMenuFlyoutItem : Windows.UI.Xaml.Controls.MenuFlyoutI
 
     static Windows.UI.Xaml.DependencyProperty IsCheckedProperty{ get; };
     static Windows.UI.Xaml.DependencyProperty GroupNameProperty{ get; };
+
+    [MUX_DEFAULT_VALUE("false")]
+    [MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnAreCheckStatesEnabledPropertyChanged")]
+    static Windows.UI.Xaml.DependencyProperty AreCheckStatesEnabledProperty{ get; };
```

Comment:
If the API is signed off, then great, if not, please mark these as preview.

Reply:
It has been reviewed!